doc/en/developer/source/release-guide/index.rst

```diff
@@ -250,19 +260,19 @@ Post the Documentation
 
    #. Add the lines::
 
-        ProxyPass /a.b.x/ http://ares.boundlessgeo.com/geoserver/a.b.x/doc/
-        ProxyPassReverse /a.b.x/ http://ares.boundlessgeo.com/geoserver/a.b.x/doc/
+        ProxyPass /a.b.x/ https://build.geoserver.org/geoserver/a.b.x/doc/
+        ProxyPassReverse /a.b.x/ https://build.geoserver.org/geoserver/a.b.x/doc/
 
       after the last similar entry.
 
    #. Modify the lines::
 
-        ProxyPass /maintain/ http://ares.boundlessgeo.com/geoserver/2.8.x/doc/
-        ProxyPassReverse /maintain/ http://ares.boundlessgeo.com/geoserver/2.8.x/doc/
-        ProxyPass /stable/ http://ares.boundlessgeo.com/geoserver/2.9.x/doc/
-        ProxyPassReverse /stable/ http://ares.boundlessgeo.com/geoserver/2.9.x/doc/
-        ProxyPass /latest/ http://ares.boundlessgeo.com/geoserver/master/doc/
-        ProxyPassReverse /latest/ http://ares.boundessgeo.com/geoserver/master/doc/
+        ProxyPass /maintain/ https://build.geoserver.org/geoserver/2.8.x/doc/
+        ProxyPassReverse /maintain/ https://build.geoserver.org/geoserver/2.8.x/doc/
+        ProxyPass /stable/ https://build.geoserver.org/geoserver/2.9.x/doc/
+        ProxyPassReverse /stable/ https://build.geoserver.org/geoserver/2.9.x/doc/
+        ProxyPass /latest/ https://build.geoserver.org/geoserver/master/doc/
+        ProxyPassReverse /latest/ https://build.geoserver.org/geoserver/master/doc/
```

Comment:
This is not yet the case, due to an SSL issue between docs.geoserver.org and build.geoserver.org. That should be fixed soon, so we should hold off merging until then.